### ISSUE TITLE
Accessibility: <html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by ericahagle** [ericahagle/fitlit#94]
![Image](https://github.com/ericahagle/fitlit/assets/133910120/aaae41d8-0bf6-4306-bc90-374c1c4b68f5)
